packages/wdio-config/package.json

```diff
@@ -29,6 +29,9 @@
     "url": "https://github.com/webdriverio/webdriverio/issues"
   },
   "dependencies": {
+    "@cucumber/messages": "22.0.0",
+    "@cucumber/gherkin": "26.2.0",
+    "@cucumber/tag-expressions": "5.0.1",
```

Comment:
Let's not have Cucumber dependencies in this package

Reply:
> Let's make sure we keep Cucumber related functionality in the Cucumber package.

I understand and agree with the sentiment, however, please acknowledge that it does not seem possible to implement this feature in the `@wdio/cucumber-framework` package.

The amount of workers to spawn is decided in `wdio-config`.
The Cucumber Adapter then does the init() followed by run() for each pre-determined worker.

There is no filtering that can be done in the framework adapter because the amount of workers as been decided already.

Please advise.

Reply:
Would dynamically importing the dependencies, according to need (config.framework) be an optimal solution here?

Reply:
> There is no filtering that can be done in the framework adapter because the amount of workers as been decided already.

It is not important how many workers are being booted. If the [`hasTest`](https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-cucumber-framework/src/index.ts#L144-L146) returns `false` then the worker closes down without being visible in the reporting. Can you use these interfaces to implement it?